services/common/src/components/projectSummary/LegalLandOwnerInformation.tsx

```diff
@@ -104,7 +104,7 @@ export const LegalLandOwnerInformation: FC = () => {
                 label="Legal Land Owner Name"
                 component={RenderField}
                 required={!is_legal_land_owner}
-                validate={!is_legal_land_owner ? [required] : []}
+                validate={!is_legal_land_owner ? [required, maxLength(100)] : []}
```

Comment:
For this one and the one below- we only want the required validation when they're not the legal land owner, but shouldn't the other methods be in the second array? Like if it's not required, it should still have a max length, and below even if it's not required should still follow email validation. Existing issue I guess but may as well fix it?